clients/drcachesim/CMakeLists.txt

```diff
@@ -499,6 +499,17 @@ endif ()
 # Be sure to give the targets qualified test names ("tool.drcache*...").
```

Comment:
> No description provided.

Please fix the PR title and description.  Normally you'd squash local commits into one prior to making the PR (once the branch is shared don't edit shared commits -- but clean up private commits prior to sharing) which would then auto-populate the PR title and description with what we want.

Reply:
Done.